prow/cmd/deck/static/prow/prow.ts

```diff
@@ -652,16 +653,22 @@ function redraw(fz: FuzzySearch): void {
 function createRerunCell(modal: HTMLElement, rerunElement: HTMLElement, prowjob: string): HTMLTableDataCellElement {
     const url = `https://${window.location.hostname}/rerun?prowjob=${prowjob}`;
     const c = document.createElement("td");
-    const i = icon.create("refresh", "Show instructions for rerunning this job");
-    i.onclick = () => {
-        modal.style.display = "block";
-        rerunElement.innerHTML = `kubectl create -f "<a href="${url}">${url}</a>"`;
-        const copyButton = document.createElement('a');
-        copyButton.className = "mdl-button mdl-js-button mdl-button--icon";
-        copyButton.onclick = () => copyToClipboardWithToast(`kubectl create -f "${url}"`);
-        copyButton.innerHTML = "<i class='material-icons state triggered' style='color: gray'>file_copy</i>";
-        rerunElement.appendChild(copyButton);
-    };
+    let i;
+    if (rerunCreatesJob) {
+      i = icon.create("refresh", "Rerun this job");
+      i.href = `${url}`;
```

Comment:
I'd prefer to see this as an asynchronous call rather than a page reload.

Reply:
But the page has to be reloaded anyways to see the new job or is there some other way? (Feel free to point my some doc, this is far from my area of expertise)